web_widget_table/static/css/web_widget.css

```
@@ -0,0 +1,6 @@
+.oe_form_field_table
+{
+    margin: 0em .2em .2em 0em;
+    display: inline-block !important;
```

```suggestion
    display: inline-block;
```

Please check if there's any way to remove the `!important` from here. Maybe you can just add a more specific selector so normal CSS priority rules apply.

It's a good practice to avoid these, because later is too hard to override in a theme or submodule.